src/fragments/lib/datastore/react-native/getting-started/30_platformIntegration.mdx

```diff
@@ -28,10 +22,9 @@ React Native CLI apps can now use SQLite with DataStore. SQLite offers considera
 To enable SQLite with DataStore for enhanced local database performance, follow the steps below:
 
 ```sh
-npx react-native@0.68.2 init AmplifyDataStoreRN --version 0.68.2
+npx react-native init AmplifyDataStoreRN
 cd AmplifyDataStoreRN
-npx amplify-app@latest
-npm install aws-amplify amazon-cognito-identity-js @aws-amplify/datastore-storage-adapter react-native-sqlite-storage @react-native-community/netinfo @react-native-async-storage/async-storage core-js
+npm install aws-amplify @aws-amplify/react-native @react-native-community/netinfo @aws-amplify/datastore-storage-adapter react-native-sqlite-storage @react-native-async-storage/async-storage react-native-get-random-values
```

Comment:
This sounds like an additional step to "Start with the [React Native CLI]" should it only list how to install `@aws-amplify/datastore-storage-adapter` here?

Reply:
That does make a lot more sense to me. This is what was there before, with everything above plus the added `@aws-amplify/datastore-storage-adapter react-native-sqlite-storage`. If I have time after migration guide I will throw that in.